spec/api/v1/transfers_spec.rb

```diff
@@ -62,222 +26,219 @@ def valid_form_a_mat_transfer_parameters
       end
 
       it "returns an Unauthorized error" do
-        get "/api/v1/projects/transfers", headers: {ApiKey: "testkey"}
+        post "/api/v1/projects/transfers", headers: {ApiKey: "testkey"}
         expect(response.body).to eq({error: "Unauthorized. Invalid or expired token."}.to_json)
-        expect(response.status).to eq(401)
       end
     end
+  end
 
-    context "when there is a valid api key in the header" do
+  describe "get requests are not allowed" do
+    context "when there is a valid api key" do
       before do
         ApiKey.create(api_key: "testkey", expires_at: Date.tomorrow)
       end
 
-      it "returns Not Allowed" do
+      it "returns Not Allowed for transfers" do
         get "/api/v1/projects/transfers", headers: {Apikey: "testkey"}
         expect(response.body).to eq({error: "405 Not Allowed"}.to_json)
         expect(response.status).to eq(405)
       end
+
+      it "returns Not Allowed for fotm a MAT transfers" do
```

Comment:
typo "form"

Reply:
This is actually in a different bit of work that this was based on! I'll fix it in another PR! 👍